hypertrace-core-graphql-platform/build.gradle.kts

```diff
@@ -16,7 +16,7 @@ dependencies {
     api("org.hypertrace.core.grpcutils:grpc-client-utils:0.8.2")
     api("org.hypertrace.core.grpcutils:grpc-client-rx-utils:0.8.2")
     api("org.hypertrace.core.grpcutils:grpc-client-rx-utils:0.8.2")
-    api("org.hypertrace.gateway.service:gateway-service-api:0.2.15")
+    api("org.hypertrace.gateway.service:gateway-service-api:0.2.20-SNAPSHOT")
```

Comment:
related?

Reply:
Ah so it's unrelated, you're just piggy backing two changes together? Won't claim I've never done it, but try to discuss both in the pr title/description.